tests/OfxParser/ParserTest.php

```diff
@@ -50,35 +43,77 @@ public function testXmlLoadStringLoadsValidXml()
 		$this->assertEquals('bar', (string)$xml->foo);
 	}
 
-	public function testCloseUnclosedXmlTags()
+    /**
+     * @return array
+     */
+	public function testCloseUnclosedXmlTagsProvider()
+    {
+        return [
+            ['<SOMETHING>', '<SOMETHING>'],
+            ['<SOMETHING>foo</SOMETHING>', '<SOMETHING>foo'],
+            ['<SOMETHING>foo</SOMETHING>', '<SOMETHING>foo</SOMETHING>'],
+            ['<BANKID>XXXXX</BANKID>', '<BANKID>XXXXX</BANKID>'],
+            ['<ACCTID>XXXXXXXXXXX</ACCTID>', '<ACCTID>XXXXXXXXXXX</ACCTID>'],
+            ['<ACCTID>-198.98</ACCTID>', '<ACCTID>-198.98</ACCTID>'],
+            ['<ACCTID>-198.98</ACCTID>', '<ACCTID>-198.98'],
+        ];
+    }
+
+    /**
+     * @dataProvider testCloseUnclosedXmlTagsProvider
+     * @param $expected
+     * @param $input
+     */
+	public function testCloseUnclosedXmlTags($expected, $input)
 	{
 		$method = new \ReflectionMethod('\OfxParser\Parser', 'closeUnclosedXmlTags');
 		$method->setAccessible(true);
 
 		$parser = new Parser();
 
-		$this->assertEquals('<SOMETHING>', $method->invoke($parser, '<SOMETHING>'));
-		$this->assertEquals('<SOMETHING>foo</SOMETHING>', $method->invoke($parser, '<SOMETHING>foo'));
-		$this->assertEquals('<SOMETHING>foo</SOMETHING>', $method->invoke($parser, '<SOMETHING>foo</SOMETHING>'));
+		$this->assertEquals($expected, $method->invoke($parser, $input));
 	}
 
-	public function testConvertSgmlToXml()
-	{
-		$sgml = <<<HERE
+	public function testConvertSgmlToXmlProvider()
```

Comment:
Please use spaces for indentation consistently, there's a mixture of tabs/spaces going on here